apps/examples/testcase/le_tc/network/tc_net_recv.c

```diff
@@ -157,16 +157,16 @@ void *recv_server(void *args)
 
 	char *msg = "Hello World !\n";
 	struct sockaddr_in sa;
-	int SocketFD = socket(PF_INET, SOCK_STREAM, IPPROTO_TCP);
+	int SOCKET_FD = socket(PF_INET, SOCK_STREAM, IPPROTO_TCP);
```

Comment:
I don't think, it's recommended to set local variables name in CAPS . Is there any issue with earlier variable name?  If you want to follow this convention , there are other variables also to change for ex ConnectFD

Reply:
Please check below on our coding rule:
```
Naming
[M12] No mixed-case, Use underscores('_') to separate words in an name.
[Example-Do]

#define TEST_VALUE	32
extern struct list_head_hash[TEST_VALUE];

int example_function(void)
{
    return(TEST_VALUE - 1);
}
[Example-Don't]

#define Test_Value	32
extern struct list_head_hash[Test_Value];

int ExampleFunction(void)
{
    return(Test_Value - 1);
}
```

Reply:
@sunghan-chang , As I could see the above guideline, it's ok to declare preprocessors with CAPS , In my opinion, it may not be standard to declare the local variables in CAPS. What's your opinion ?

Reply:
@pradeepns85 Agree. Usually CAPS is a definition.

Reply:
I am going to modify all FD variables that violate the coding rules.

Reply:
@KimDongEon Pradeep said, capital character is incorrect. And I agree his opinion. Why do you change all of variables to CAP?

Reply:
There was a mistake. I changed everything to lower case.